migrations/sql/V2020.11.09.1222__AddCoverageSetMetadata.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE coverage_set_metadata
+(
+    "id"          INTEGER   NOT NULL,
+    "uploaded_by" TEXT      NOT NULL,
+    "uploaded_on" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "filename"    TEXT      NULL,
```

Comment:
leaving this optional as not sure how to get it from the multipart file upload and it's more of a nice to have, so will probably only be stored in a future iteration